RuntimeUnityEditor.Bepin5/PatchInspector/PatchInspector.cs

```diff
@@ -0,0 +1,1375 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Reflection;
+using System.Reflection.Emit;
+using System.Text;
+using BepInEx;
+using HarmonyLib;
+using RuntimeUnityEditor.Core;
+using RuntimeUnityEditor.Core.Utils.Abstractions;
+using UnityEngine;
+
+namespace RuntimeUnityEditor.Bepin5.PatchInspector
+{
+	public class PatchInspector : Window<PatchInspector>
```

Comment:
Need to add `Instance.AddFeature(new PatchInspector.PatchInspector());` in bep5 Start or it won't load.